src/pyqt_tools/__main__.py

```diff
@@ -0,0 +1,19 @@
+""" Enables support for calling entrypoints using `python -m` or `pythonw -m`.
+On windows using `pythonw -m pyqt6_tools designer` in a shortcut allows opening
+QtDesigner without showing a command prompt window that stays open the entire
+time that QtDesigner is open.
+"""
+
+
+import importlib
+import pathlib
+import sys
+
+if __name__ == "__main__":
+    # Relative imports don't work when called by `python -m ...`, get the name
```

Comment:
I'm confused.  No package that uses relative imports can be run as a module?  That doesn't sound right.

Reply:
Ah, you are right, I'm thinking of when you try to pass a .py file to python.
```
C:\Users\example>py "C:\Program Files\Python39\Lib\site-packages\pyqt5_tools\__main__.py"
Traceback (most recent call last):
  File "C:\Program Files\Python39\Lib\site-packages\pyqt5_tools\__main__.py", line 10, in <module>
    from . import entrypoints
ImportError: attempted relative import with no known parent package
```

That simplifies this code a lot. I copied the boilerplate for this from some of my own `__main__.py` files, which I'll probably revisit in the near future.